.travis.yml

```diff
@@ -14,35 +14,31 @@ env:
       ENGINE=docker
     - ACTION=bandit
       OS=fedora
-      OS_VERSION=29
+      OS_VERSION=31
       PYTHON_VERSION=3
       ENGINE=docker
     - ACTION=pylint
       OS=fedora
-      OS_VERSION=29
-      PYTHON_VERSION=2
-      ENGINE=docker
-    - ACTION=pylint
-      OS=fedora
-      OS_VERSION=29
+      OS_VERSION=31
       PYTHON_VERSION=3
       ENGINE=docker
     - OS=centos
       OS_VERSION=7
       PYTHON_VERSION=2
       ENGINE=docker
     - OS=fedora
-      OS_VERSION=29
-      PYTHON_VERSION=2
-      ENGINE=docker
-    - OS=fedora
-      OS_VERSION=29
+      OS_VERSION=31
       PYTHON_VERSION=3
       ENGINE=docker
     - OS=fedora
       OS_VERSION=30
       PYTHON_VERSION=3
       ENGINE=docker
+    - ACTION=pylint
+      OS=fedora
+      OS_VERSION=29
+      PYTHON_VERSION=2
+      ENGINE=docker
```

Comment:
Could this be moved next to the other `pylint` action? It would be nice to have them grouped together.

Reply:
I am wondering if we really want this here. Fedora 29 Has reached its EOL in the end of 2019. IMHO, we should just stop running the CI in F29. If this means no more py2 tests in Fedora, so be it. We can keep running py2 tests in CentoOS7. What do you think?

Reply:
I'd keep it there until we defer py2 support. It's just linting and newer versions of pylint stopped supporting py2. So F29 is the most recent we can use for py2 linting